extensions/guacamole-auth-cas/src/main/resources/guac-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
 
-    "guacamoleVersion" : "1.2.0",
+    "guacamoleVersion" : "1.3.0",
```

Comment:
I'm not sure this should be done as part of this change. It definitely needs to be done for the overall 1.3.0 release, but it seems out-of-scope, here.